src/app/searchResults/searchResults.ministries.js

```diff
@@ -121,8 +115,8 @@ export default [
   },
   {
     name: 'Unto™',
```

Comment:
Is the `™` desired? It must have blended in with the quotes when I was looking at it before and I didn't notice it.

Reply:
I believe it is.